cardano-api/src/Cardano/Api/Fees.hs

```diff
@@ -1039,12 +1080,17 @@ calculateMinimumUTxO era txout@(TxOut _ v _ _) pparams' =
     ShelleyBasedEraAllegra -> calcMinUTxOAllegraMary
     ShelleyBasedEraMary -> calcMinUTxOAllegraMary
     ShelleyBasedEraAlonzo ->
-      case protocolParamUTxOCostPerWord pparams' of
-        Just (Lovelace costPerWord) -> do
-          Right . lovelaceToValue
-            $ Lovelace (Alonzo.utxoEntrySize (toShelleyTxOutAny era txout) * costPerWord)
-        Nothing -> Left PParamsUTxOCostPerWordMissing
-    ShelleyBasedEraBabbage -> error "TODO: Babbage"
+      let lTxOut = toShelleyTxOutAny era txout
+          babPParams = toAlonzoPParams pparams'
+          minUTxO = Shelley.evaluateMinLovelaceOutput babPParams lTxOut
+          val = lovelaceToValue $ fromShelleyLovelace minUTxO
+      in Right val
```

Comment:
I notice that this changed and that the new version doesn't fail.  What's the story here?

Reply:
The ledger provides an instance called `CLI` with a function `evaluateMinLovelaceOutput`. I think after I implemented the logic manually here they created that class that duplicates the logic.